sdk/Sdk.Generators/DiagnosticDescriptors.cs

```diff
@@ -62,5 +62,12 @@ private static DiagnosticDescriptor Create(string id, string title, string messa
                     messageFormat: "Invalid use of a retry attribute. Check that the attribute is used on a trigger that supports function-level retry.",
                     category: "FunctionMetadataGeneration",
                     severity: DiagnosticSeverity.Error);
+        public static DiagnosticDescriptor InvalidBindingAttributeArgument { get; }
```

Comment:
Named arguments can be null ([example](https://github.com/Azure/azure-functions-dotnet-worker/blob/b4e4ba87f09ce6e7d157d9ed02661bc9db4f4d52/sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs#L700)).

This error deals specifically when an arg can't be parsed using [this method](https://github.com/Azure/azure-functions-dotnet-worker/blob/b4e4ba87f09ce6e7d157d9ed02661bc9db4f4d52/sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs#L700). Possibilities are that arg is not of a supported type, arg is null, or it's an enum that couldn't be parsed.